WalletWasabi.Gui/Validation/Validations.cs

```diff
@@ -0,0 +1,101 @@
+using ReactiveUI;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Gui.Validation
+{
+	public class Validations : ReactiveObject, IRegisterValidationMethod, IValidations
+	{
+		public Validations()
+		{
+			ErrorsByPropertyName = new Dictionary<string, ErrorDescriptors>();
+			ValidationMethods = new Dictionary<string, ValidateMethod>();
+		}
+
+		private Dictionary<string, ErrorDescriptors> ErrorsByPropertyName { get; }
+
+		private Dictionary<string, ValidateMethod> ValidationMethods { get; }
+
+		public event EventHandler<DataErrorsChangedEventArgs> ErrorsChanged;
```

Comment:
This event should be moved above the two properties.
Maybe this is what causes the CodeFactor issue.

Reply:
done.